modules/data.land/R/extract_NEON_veg.R

```diff
@@ -10,16 +10,12 @@
 #' @param neonsites prepared datasets table from NEON using neonstore::neon_sites(api = "https://data.neonscience.org/api/v0", .token = Sys.getenv("NEON_TOKEN"))
 #' @param ... Additional parameters
 #' 
-#'
+#' 
 #' @return veg_info object to be passed to extract_veg within ic_process
 #' @author Alexis Helgeson and Michael Dietze
+#'
 #' @export
-#' 
-#' @importFrom rlang .data
```

Comment:
Since the code below still uses `.data` (see e.g. line 49), this `@importFrom` should not be removed.

Reply:
Yep this was a mistake not sure how this got deleted.